test/e2e_node/container_manager_test.go

```diff
@@ -86,6 +88,15 @@ var _ = Describe("Kubelet Container Manager", func() {
 				err := cl.Pods(namespace).Delete(podName, &api.DeleteOptions{})
 				Expect(err).To(BeNil(), fmt.Sprintf("Error deleting Pod %v", err))
 			})
+
+			AfterEach(func() {
+				cl.Pods(namespace).Delete(podName, &api.DeleteOptions{})
+				Eventually(func() error {
+					_, err := cl.Pods(namespace).Get(podName)
```

Comment:
Not sure what 'container-manager' tries to test, I'll check if the error equals to 'IsNotFound'.